docs/docs/building-a-blog.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Building a blog
+issue: https://github.com/gatsbyjs/gatsby/pull/6245
```

Comment:
Stubs don't link to closed PRs, since there isn't a clear process for what a contributor should do with it. A better approach would be to file individual issues for each of these stubs so the scope of each one is limited to a single doc and people can pick them up one-by-one. If you have time to do that, it would be amazing and here's an example of it! https://github.com/gatsbyjs/gatsby/issues/18281 

Outside of that, I don't think we should merge this as-is with links to an old PR. So let us know what you have time to do!